pkg/installer/src/utils.rs

```diff
@@ -88,15 +88,32 @@ pub fn save_config_value(c: &mut Cursive, k: &str, v: &str, m: bool) -> crate::e
     Ok(())
 }
 
-pub fn edit_config_value(c: &mut Cursive, k: &str, v: &str) -> crate::error::Result<()> {
+pub fn add_config_value(c: &mut Cursive, k: &str, v: &str) -> crate::error::Result<()> {
     let mut s: GlobalState = c.take_user_data().unwrap();
-    let mut vedit = s.data.map.get(k).unwrap().clone();
-    if vedit == "_____" {
-        vedit = v.to_string();
+    let mut vadd = s.data.map.get(k).unwrap().clone();
```

Comment:
can we be sure we can call unwrap() here? if `k` doesn't exist in the map we panic

Reply:
The map is initialized when the program starts. So, we always have values.